view.py

```diff
@@ -681,7 +685,11 @@ def on_done(s):
                         self.update_needed += 1
                     def onready():
                         sublime.set_timeout(self.update, 0)
-                    _, ready = neo.vim.press('<cr>', onready)
+                        
+                    try:
+                        _, ready = neo.vim.press('<cr>', onready)
+                    except Exception:
+                        ready = True
```

Comment:
If an error occurs it's probably not "ready", is it?

More generally, it might be a good idea to show an error (with backtrace) to the user, or at least log it.

Reply:
I don't know why, but it always has no response after error thrown from this three place. I'm not good at python, I just want give some clue. by the way, I haven't crashed again after add them.